docs/output-kafka.asciidoc

```diff
@@ -248,6 +249,21 @@ the timeout setting for initial metadata request to fetch topic metadata.
 
 the max time in milliseconds before a metadata refresh is forced.
 
+[id="plugins-{type}s-{plugin}-partitioner"]
+===== `partitioner`
+
+* Value type is <<string,string>>
+* There is no default value for this setting.
+
+The default behavior is to hash the `message_key` of an event to get the partition.
+When no message key is present, the plugin picks a partition in a round-robin fashion.
+
+Available options for choosing a partitioning strategy are as follows:
+
+* `default` use the default partitioner as described above
+* `round_robin` distributes writes to all partitions equally, regardless of `message_key`
+* `uniform_sticky` sticks to a partition for the duration of a batch than randomly picks a new one
+
```

Comment:
thx - though about that, wasn't sure as user would need to add to class-path somehow using a custom class ... than the client .jar might get updated later and contain more partitioner impls.

Reply:
That's a good point - it is something we do in other plugins - the `jdbc` plugin can either take a path to an external jar, or it can be placed in the classpath. We can either not document this feature at all, or tell users to place the jar featuring their custom partitioner in the classpath for now

Reply:
Okay - let's leave it as is and we'll see if someone comes up with such requirement. 
Also, while I do not mind setting Java specific class names it seems a bit ugly that the plugin is exposing class names by default e.g. 
`value_serializer => 'org.apache.kafka.common.serialization.StringSerializer'`
instead of having a supported (and documented): `value_serializer => 'string'`

I'll check if we can update those quickly in a follow-up PR